test/system/test_tpm2_createpolicy.sh

```diff
@@ -50,16 +50,16 @@ for halg in sha1 sha256
 do
     cleanup
 
-    #
-    # This script expects PCR index 0 to be equal to 0x03
-    #
-    value=`tpm2_pcrlist -L $halg:0 | grep PCR_00 | cut -d\: -f 2-`
-    if [[ $value -ne 0x03 ]]; then
-        echo "Expected PCR 0 \"$halg\" bank to be 0x3, found: \"$value\"" 1>&2
-        exit 1
-	fi
+    # Create file containing expected PCR value
+    if [ $halg = sha1 ]; then
+        digestlength=20
+    else
+        digestlength=32
+    fi
```

Comment:
this is brittle. I would look it up in the hash table and use wc -c to get the length.

Reply:
Not that brittle, since the possible values for $halg are defined just four lines above, but if you can tell me where to find this hash table I can change the algorithm.

Reply:
@webmeister maybe something like this would be nicer?

```
declare -A digestlengths=(["sha1"]=20 ["sha256"]=32)
...

for halg in ${!digestlengths[@]}
do
   ...
    head -c $((${digestlengths[$halg]} - 1)) /dev/zero > pcr.in
    ...
done
```

Reply:
Thanks, that keeps the values nicely together. I'll update the patch accordingly.